public/app/features/alerting/unified/components/rules/RulesTable.tsx

```diff
@@ -1,4 +1,6 @@
 import { css, cx } from '@emotion/css';
+import { isBefore } from 'date-fns';
+import formatDuration from 'date-fns/formatDuration';
```

Comment:
Any particular reason why we're not also destructuring this like the import above?

Reply:
No reason, fixed 👍